website/content/partials/known-issues/kv2-url-change.mdx

```diff
@@ -0,0 +1,25 @@
+### URL change for KV v2 plugin ((#kv2-url-change))
+
+#### Affected versions
+
+1.15.0+
+
+#### Issue
+
+Recent improvements to Vault included improvements and refactoring to the KV v2
+engine that affect existing plugin URLs.
```

Comment:
I think we need to clarify that we're talking about the Vault UI changes.  

Something like...

```suggestion
Recent improvements to the Vault UI updated the URL structure of the KV v2 secrets engine that affect existing URLs.
```